lib/internal/Magento/Framework/View/Asset/MergeStrategy/Direct.php

```diff
@@ -30,16 +30,23 @@ class Direct implements \Magento\Framework\View\Asset\MergeStrategyInterface
      */
     private $cssUrlResolver;
 
+    /**
+     * @var \Magento\Framework\Math\Random
+     */
+    protected $mathRandom;
+
     /**
      * @param \Magento\Framework\Filesystem $filesystem
      * @param \Magento\Framework\View\Url\CssResolver $cssUrlResolver
      */
     public function __construct(
         \Magento\Framework\Filesystem $filesystem,
-        \Magento\Framework\View\Url\CssResolver $cssUrlResolver
+        \Magento\Framework\View\Url\CssResolver $cssUrlResolver,
+        \Magento\Framework\Math\Random $mathRandom
```

Comment:
Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add required parameters to the constructor. Please look **Adding a constructor parameter** section to do it in the correct way.

Reply:
Resolved in latest commit.